lua/notify/stages/util.lua

```diff
@@ -142,12 +142,17 @@ end
 function M.slot_after_previous(win, open_windows, direction)
   local key = slot_key(direction)
   local cmp = is_increasing(direction) and less or greater
-  local exists, cur_win_conf = pcall(vim.api.nvim_win_get_config, win)
+  local exists, cur_win_conf
+  if vim.version().minor < 10 then
+    exists, cur_win_conf = pcall(vim.api.nvim_win_get_config, win)
+  else
+    exists, cur_win_conf = util.get_win_config(win)
+  end
```

Comment:
```suggestion
local exists, cur_win_conf = pcall(vim.api.nvim_win_get_config, win)
```
I would also suggest to leave this as it was, since it doesn't affect the problem at hand.

Reply:
Of course @kawre idea is good and I'll take time to implement it tomorrow.

> I would also suggest to leave this as it was, since it doesn't affect the problem at hand.

The only reason I did this is because there is already an util function that does the same thing, so I thought it was a good idea to reuse what already exists.

Reply:
Yes, but the util function also uses a `pcall` to `vim.api.nvim_win_get_config` under the hood as well, so it's essentially the same thing. And both basically return the window handler at hand. Also there's no need to make a check here for the Neovim version. Only at the places where `key` is existent.